release-notes/azure/v11.0.0.md

```diff
@@ -75,3 +75,6 @@ This is the first Giant Swarm release which includes Kubernetes v1.16. In additi
 ### node_exporter v0.18.1 ([Giant Swarm app v1.2.0](https://github.com/giantswarm/node-exporter-app/blob/master/CHANGELOG.md#120-2020-01-08))
 - Updated to upstream version 0.18.1 - [changelog](https://github.com/prometheus/node_exporter/blob/master/CHANGELOG.md#0181--2019-06-04).
 - Changed priority class to `system-node-critical`.
+
+### nginx-ingress-controller v0.26.1 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v111-2020-01-04))
```

Comment:
Would be nice to have it above `node_exporter` to keep alphabetical order

Reply:
Ah, I didn't realize they were alphabetical.